packages/devtools_app/test/memory_screen_test.dart

```diff
@@ -368,12 +370,40 @@ void main() {
       await tester.pumpAndSettle(_twoSeconds);
     }
 
+    Future<void> checkGolden(String goldenFilename, {Key key}) async {
+      // Await delay for golden comparison.
+      await tester.pumpAndSettle(const Duration(seconds: 2));
+
+      final finder = key == null ? find.byType(MemoryBody) : find.byKey(key);
+
+      // Screenshot should display left-side tree table fully expanded and the monitor
+      // allocation leaf node 'Monitor <timestamp>' selected. The right-side displaying
+      // all allocations in a flat table, no items checked (tracked), search should
+      // be enabled with focus. No tree table displayed on the bottom only empty message.
+      await expectLater(finder, matchesGoldenFile(goldenFilename));
+    }
+
     await pumpAndSettleTwoSeconds();
 
     await pumpMemoryScreen(tester);
 
+    // Any exported memory files in the /tmp or $TMPDIR or /var/folders/Downloads will cause
+    // the memory page's drop-down widget width to be wider than normal (longest exported file
+    // name). For goldens don't generate snapshots as they we be slightly different on the bots.
     expect(
-        find.byKey(HeapTreeViewState.dartHeapAnalysisTabKey), findsOneWidget);
+      controller.memoryLog.offlineFiles().isEmpty,
+      isTrue,
+      reason:
+          '\n\n=========================================================================\n'
```

Comment:
nit: little odd to start the reason with `\n\n`

Reply:
Did it this was because its the very first failure if it happens a lot of text above and below kind of obscuring so I wanted to make it stand out as its bad.  Can certainly remove.  If there's a strong opinion I'll remove.